lib/templ.ml

```diff
@@ -118,20 +118,71 @@ let setup_link conf =
 let esc s = (Util.escape_html s :> string)
 
 let rec eval_variable conf =
+  let urlv_aux evar str =
```

Comment:
I don't understand what it is doing, can you add a small comment on what it does?

Reply:
merci